message/error.go

```diff
@@ -15,7 +16,13 @@ type errorHandler struct {
 	err error
 }
 
-func (a *errorHandler) Do(_ context.Context, _ Handler, _ *servicebus.Message) Handler {
-	// can trace the error before abandoning once we settle on a tracing system that can be passed via context
+func (a *errorHandler) Do(ctx context.Context, _ Handler, msg *servicebus.Message) Handler {
+	ctx, span := startSpanFromMessageAndContext(ctx, "go-shuttle.errorHandler.Do", msg)
+	defer span.End()
+
+	if a.err != nil {
```

Comment:
unlike the ones above this is less important to log since it comes from the user and they can log it if they want but doesn't hurt to double log probably.

Reply:
It makes sense. It's not necessary to log the error here.